### DISCUSSION TITLE
HACS warning chiamate deprecate

### DISCUSSION THREAD

**Discussion by StefanoPirazzi** [indomus/forum#3139] [HUB: Home Assistant]
Salve ho installato HACS 1.34.0 su Homeassistant OS (Core 2024.7.1 Supervisor 2024.06.2 Operating System 12.4)   installata su Raspberry PI 4.
Al riavvio nel LOG mi compaiono i seguenti Warning ...

Registratore: homeassistant.helpers.frame
Fonte: helpers/frame.py:184
Prima occorrenza: 17:03:57 (4 occorrenze)
Ultima registrazione: 17:03:57

    Detected that custom integration 'hacs' calls hass.http.register_static_path which is deprecated because it does blocking I/O in the event loop, instead call `await hass.http.async_register_static_paths([StaticPathConfig("/hacsfiles/frontend", "/config/custom_components/hacs/hacs_frontend_experimental", False)])`; This function will be removed in 2025.7 at custom_components/hacs/frontend.py, line 49: hass.http.register_static_path(, please create a bug report at https://github.com/hacs/integration/issues

    Detected that custom integration 'hacs' calls hass.http.register_static_path which is deprecated because it does blocking I/O in the event loop, instead call `await hass.http.async_register_static_paths([StaticPathConfig("/hacsfiles/iconset.js", "/config/custom_components/hacs/iconset.js", True)])`; This function will be removed in 2025.7 at custom_components/hacs/frontend.py, line 57: hass.http.register_static_path(, please create a bug report at https://github.com/hacs/integration/issues

    Detected that custom integration 'hacs' accesses hass.components.frontend. This is deprecated and will stop working in Home Assistant 2024.9, it should be updated to import functions used from frontend directly at custom_components/hacs/frontend.py, line 68: hass.components.frontend.async_register_built_in_panel(, please create a bug report at https://github.com/hacs/integration/issues

    Detected that custom integration 'hacs' accesses hass.helpers.event. This is deprecated and will stop working in Home Assistant 2024.11, it should be updated to import functions used from event directly at custom_components/hacs/base.py, line 642: self.hass.helpers.event.async_track_time_interval(, please create a bug report at https://github.com/hacs/integration/issues

Come posso risolvere il problema?

Stefano

**Comment by KC-inDomus** [comment]
Aspettando che esca l'aggiornamento di hacs 👍

**Comment by Golem3G** [comment]
Buon pomeriggio, ho lo stesso problema da due settimane. Ho disistallato e intallato di nuovo hacs ma non risolvo. Come posso fare?

**Reply by snakuzzo** [reply]
Risolvi come è già stato detto l'anno scorso

**Reply by Golem3G** [reply]
mi mandi un link

**Reply by snakuzzo** [reply]
https://github.com/indomus/forum/discussions/3139#discussioncomment-9984202
